lib/cb/clients/recommendation.rb

```diff
@@ -65,11 +65,10 @@ def self.for_user(*args)
       def self.for_company(company_did)
         my_api = Cb::Utils::Api.instance
         json_hash = my_api.cb_get(Cb.configuration.uri_recommendation_for_company, query: { CompanyDID: company_did })
-
         jobs = []
         if json_hash.key?('Results')
           if json_hash['Results'].key?('JobRecommendation')
-            json_hash['Results']['JobRecommendation']['Jobs'].each do |cur_job|
+            json_hash['Results']['JobRecommendation']['Jobs']['CompanyJob'].each do |cur_job|
```

Comment:
You should also do it for the `"CompanyJob"`. I just tested it, and calling `#each` on a `NilClass` is no bueno either.